rpm/oidc-agent.spec

```diff
@@ -104,7 +104,7 @@ Requires: webkit2gtk3
 Requires: gtk3
 %else
 #Requires: webkit2gtk3-minibrowser
-Requires: webkit2gtk3
+Requires: webkitgtk3
```

Comment:
3 or 4? Or does it not matter?

Reply:
4, it was an error that `make dockerised_test_rpms` found :)